tests/unit/info-keysizes.tcl

```diff
@@ -57,7 +57,7 @@ proc run_cmd_verify_hist {cmd expOutput {waitCond 0} } {
     
     # If we are testing `replicaMode` then need to wait for the replica to catch up    
     if {$replicaMode eq 1} {        
-         wait_for_condition 50 20 {
+         wait_for_condition 50 50 {
```

Comment:
use `wait_for_ofs_sync` to wait for master/slave synchronization to complete?

Reply:
either way, it makes busy wait till condition fullfils. I wouldn't bother to change (as it is already written and working..)